src/renault_api/cli/renault_vehicle.py

```diff
@@ -203,16 +207,21 @@ async def update_battery_status(
         ):
             response.chargingStatus = 0.0
 
-        items = [
-            ("Battery level", response.batteryLevel, "%"),
-            ("Last updated", response.timestamp, "tzdatetime"),
-            ("Available energy", response.batteryAvailableEnergy, "kWh"),
-            ("Range estimate", response.batteryAutonomy, "km"),
-            ("Plug state", response.get_plug_status(), None),
-            ("Charging state", response.get_charging_status(), None),
-            ("Charge rate", response.chargingInstantaneousPower, "kW"),
-            ("Time remaining", response.chargingRemainingTime, "min"),
-        ]
+        if ctx_data["json"]:  # pragma: no cover
```

Comment:
Please remove no cover here also
```suggestion
        if ctx_data["json"]:
```